tests/test_georaster_tiling.py

```diff
@@ -435,6 +435,30 @@ def test_crop_raises_error_for_impossible_transformation(self):
         with self.assertRaises(GeoRaster2Error):
             raster.crop(vector)
 
+    def test_crop_of_rasters_with_opposite_affine_and_data_return_the_same(self):
```

Comment:
Doesn't this belong to #79?

Reply:
I'm using the same code as in #79 to test that we are getting the same cropped raster, and because we have issue #79 I need to compare `r1c.to_png() == r2c.to_png()` instead of `r1c == r2c`.